### DISCUSSION TITLE
Add subtle glow to Ahab's spear/harpoons

### DISCUSSION THREAD

**Discussion by leonard84** [FakeFishGames/Barotrauma#8636] [Ideas and suggestions]
Currently, it is quite finicky and time consuming to retrieve all fired harpoons.
In case of regular harpoons, I would just leave them but for Ahab's spear it is quite painful given how hard it is to get them in the first place. It would be cool if they had a subtle glow to them, not enough so that you could use them as light source, but enough so that you can see where they are lying on the ground.

Alternatively, the ropes should actually pull the spears back to into your inventory. This could be balanced by making you have to press a button and not doing anything else, so that you can't use this during combat.

**Comment by ghost** [comment]
Perhaps make harpoons buoyant aswell as the glow? they wouldn't sink to the bottom of the ocean, and they'd be easier to spot.

**Reply by mLuby** [reply]
I like the glow idea, maybe even on the containing cabinet. Neutrally buoyant spears could work too, though it might throw off their launch trajectory and they might keep flying quite far. Another idea is to make the spears (barely) visible on sonar. Or always keep the rope attached to these spears and have them reel back in after a few seconds. BTW all of these could be modded in (I know it's a suggestion for the base game though).

**Comment by AshEdwardsDesign** [comment]
+1 for the glow idea at a minimum.
